src/mlpack/bindings/markdown/print_docs.cpp

```diff
@@ -213,37 +219,44 @@ void PrintDocs(const std::string& bindingName,
     cout << "{: #" << languages[i] << "_" << bindingName
         << "_detailed-documentation }" << endl;
     cout << endl;
-    string doc = boost::replace_all_copy(programDoc.documentation(),
+    string desc = boost::replace_all_copy(longDescription.longDescription(),
                                          "|", "\\|");
-    cout << doc << endl;
+    cout << desc << endl;
```

Comment:
Should we also use two line breaks here?  I'm not sure what the right thing is, so correct me if I overlooked something and it was previously only one line break.

Reply:
Done. Thanks for the suggestion. :+1: